lib/person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :name,
+              :interests,
+              :supplies
+  def initialize(info)
+    @name = info[:name]
+    @interests = info[:interests]
+    @supplies = Hash.new(0)
+  end
+
+  def add_supply(item, quantity)
+    supplies[item] += quantity
+  end
+
+  #getting close on this one! Left off here
+  def can_build?(craft)
+      hash = @supplies.transform_keys do |key|
```

Comment:
So what we really are wanting to check here is if a person has enough supplies to build the craft being passed. So you would want to iterate over your `supplies_required` and check the keys value has enough. So a possible solution could be:
```
def can_build?(craft)
    supply_check = craft.supplies_required.map do |supply, quantity|
      @supplies[supply.to_s] >= quantity
    end
    supply_check.all?
end
```